compileall2.py

```diff
@@ -413,6 +432,10 @@ def main():
     if args.opt_levels is None:
         args.opt_levels = [-1]
 
+    if len(args.opt_levels) == 1 and args.hardlink_dupes:
+        parser.error(("Hardlinking of duplicated bytecode make sense "
+                      "only for more than one optimization level."))
```

Comment:
This seems to be handled on 2 places. Is it necessary?

Reply:
It is. The situation is the same for another incompatible combination. You can use compileall as a CLI tool and also as a module. Moreover, `parser.error` causes the help text to show up, not just an exception as `raise` does.